src/main/java/be/syntra/devshop/DevshopBack/factories/UserFactory.java

```diff
@@ -24,7 +19,7 @@ public User ofSecurity(
         return User.builder()
                 .firstName(firstName)
                 .lastName(lastName)
-                .fullName(firstName + " " + lastName)
+                .fullName(capitalizeFully(firstName + " " + lastName, ' ', '-'))
```

Comment:
If I see it correctly the two last params are chars, which we look for in the string. So we can capitalize all the words in the String. When does the String have a '-' here?

Reply:
this is an imported library. It capitalizes every name fed into it. So when someone registers PAul GeRartS, it turns it into -> Paul Gerarts. What you are referring to, are delimiters. One is indeed the spacing, the other is when names have a '-' in them. e.g. "someone name-secondpartname -> Someone Name-Secondpartname.

Reply:
oh yeah some people have "fancy" names.